docs/blog/author.yaml

```diff
@@ -312,3 +312,7 @@
   bio: Web Developer, Open Source advocate, new tech explorer - https://lannonbr.com/
   avatar: avatars/benjamin-lannon.jpg
   twitter: "@lannonbr"
+  - id: Hashim Warren
```

Comment:
Oh I think I see what's going on here–this line is indented a bit too far. Aligning it with the other `- id` lines should fix it!